js/main.js

```diff
@@ -45,75 +57,91 @@ function acceptFox() {
       thirdColdiv.appendChild(collectionImg);
       query.childNodes[1].appendChild(thirdColdiv);
-    } else if (i < 6) {
-      thirdColdiv = document.createElement('div');
-      thirdColdiv.classList.add('column-third');
-      thirdColdiv.classList.add('collection-item');
-      collectionImg = document.createElement('img');
-      collectionImg.classList.add('collection-img');
-      collectionImg.setAttribute('src', data.collection[i].foxImage);
-      thirdColdiv.appendChild(collectionImg);
-      query.childNodes[3].appendChild(thirdColdiv);
-
-    } else if (i < 7) {
-      thirdColdiv = document.createElement('div');
-      thirdColdiv.classList.add('column-third');
-      thirdColdiv.classList.add('collection-item');
-      collectionImg = document.createElement('img');
-      collectionImg.classList.add('collection-img');
-      collectionImg.setAttribute('src', data.collection[i].foxImage);
-      thirdColdiv.appendChild(collectionImg);
-      query.childNodes[5].appendChild(thirdColdiv);
-    } else {
-      break;
     }
 
   }
   return query;
 }
-*/
-function collectionLoad(query) {
 
-  for (var i = 0; i < data.collection.length; i++) {
-    if (i < 7) {
-      var thirdColdiv = document.createElement('div');
-      thirdColdiv.classList.add('column-third');
-      thirdColdiv.classList.add('collection-item');
-      var collectionImg = document.createElement('img');
-      collectionImg.classList.add('collection-img');
-      collectionImg.setAttribute('src', data.collection[i].foxImage);
-      thirdColdiv.appendChild(collectionImg);
-      query.childNodes[1].appendChild(thirdColdiv);
-    }
+function loadProfile(event) {
+  if (event.target.tagName === 'IMG') {
 
+    var profileImg = document.querySelector('.profile-img');
+    profileImg.setAttribute('src', event.target.src);
+    profileImg.setAttribute('id-number', event.target.getAttribute('id-number'));
+    profileIndex = Number(event.target.getAttribute('id-number'));
+    switchView('profile');
   }
-  return query;
 }
 
+function switchView(view) {
+  var dataViewList = document.querySelectorAll('div[data-view]');
+  for (var i = 0; i < dataViewList.length; i++) {
+    if (dataViewList[i].getAttribute('data-view') !== view) {
+      dataViewList[i].classList.add('hidden');
+    } else if (dataViewList[i].getAttribute('data-view') === view) {
+      dataViewList[i].classList.remove('hidden');
+    }
+  }
+  data.view = view;
+
+}
+
+function summonView() {
+  switchView('summon');
+}
+function collectionShow() {
+  switchView('collection');
+}
+
+function releaseModal() {
+  var modalDiv = document.querySelector('.modal-div');
+  modalDiv.classList.remove('hidden');
+}
+
+function releaseFox() {
+  data.collection.splice(profileIndex, 1);
+  var modalDiv = document.querySelector('.modal-div');
+  modalDiv.classList.add('hidden');
+  var foxImg = document.querySelectorAll('img[id-number="' + profileIndex + '"]');
+  foxImg[0].parentElement.remove();
```

Comment:
Is there a reason you are using `querySelectorAll` here? If you use `querySelector`, I think you can get around using the `[0]` index portion of this, which will simplify understanding this code.

Reply:
Yes that was to reset another image that I later realize was going to hidden anyway.